### PR TITLE
Add examples to documentation: Part-9 (cleanup)

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
Part of #1492

### CODE REVIEW

File: R/fixed_regex_linter.R [hunk 1]
```diff
@@ -10,6 +10,29 @@
 #'   likely cases. It should _never_ report false positives, however; please
 #'   report false positives as an error.
 #'
+#' @examples
+#' # will produce lints
+#' lint(
+#'   text = 'gsub("\\\\.", "", x)',
```
Comment (MichaelChirico) [inline R/fixed_regex_linter.R:16]: use writeLines, double escaping

File: R/fixed_regex_linter.R [hunk 2]
```diff
@@ -10,6 +10,29 @@
...
+#'   linters = fixed_regex_linter()
+#' )
+#'
+#' lint(
+#'   text = 'grepl("a[*]b", x)',
+#'   linters = fixed_regex_linter()
+#' )
```
Comment (MichaelChirico) [inline R/fixed_regex_linter.R:23]: add a stringr example?

**Comment by IndrajeetPatil** [comment]
@MichaelChirico Do you mind adding examples for `nonportable_path_linter()`? 

I couldn't manage to do it, and that's the only one that remains. Feel free to push in this PR itself, if you can work on it.

**Comment by MichaelChirico** [comment]
> Do you mind adding examples for nonportable_path_linter()?

Was this a matter of not understanding the linter?

I also don't understand the path linters TBH, but I can look into it if you'd like :)

File: R/fixed_regex_linter.R [hunk 3]
```diff
@@ -10,6 +10,45 @@
...
+#'
+#' code_lines <- 'stringr::str_subset(x, "\\\\$")'
+#' writeLines(code_lines)
```
Comment (MichaelChirico) [inline R/fixed_regex_linter.R:28]: WDYT about an example that doesn't use any regex special characters? e.g. `grepl("Munich", address)`

**Comment by IndrajeetPatil** [comment]
> Was this a matter of not understanding the linter?
> I also don't understand the path linters TBH, but I can look into it if you'd like :)

Partly poor understanding, and partly frustration with creating working examples. Anything and everything I tried, it either linted or produced error because escapes and what not, so decided to give up.

**Comment by MichaelChirico** [comment]
ok, let's just leave it as a follow-up. those linters need some love / refactoring anyway & are not on by default, I'm fine to proceed.

**Review by MichaelChirico — APPROVED** [review]

**Pull request merged**